app/browser/api/ledger.js

```diff
@@ -185,13 +198,12 @@ const onBootStateFile = (state) => {
   try {
     clientprep()
     client = ledgerClient(null, underscore.extend({roundtrip: roundtrip}, clientOptions), null)
-    client.publisherTimestamp((err, result) => {
-      if (err) {
-        console.error('Error while retrieving publisher timestamp', err.toString())
-        return
-      }
-      appActions.onPublisherTimestamp(result.timestamp)
-    })
+
+    getPublisherTimestamp()
+    if (verifiedTimeoutId) {
+      clearInterval(verifiedTimeoutId)
+    }
+    verifiedTimeoutId = setInterval(getPublisherTimestamp, 1 * ledgerUtil.milliseconds.hour)
```

Comment:
Do we need to call `clearInterval` and set to false when payments is turned off?

Reply:
it's just for safety, because we trigger boot via action, so there could be race problem if we trigger interval in other place

Reply:
fixed base on our slack conversation